api/src/main/java/com/theokanning/openai/completion/chat/ChatCompletionRequest.java

```diff
@@ -98,7 +98,7 @@ public class ChatCompletionRequest {
     /**
      * A list of the available functions.
      */
-    List<ChatFunction> functions;
+    List<?> functions;
```

Comment:
Would it make sense for ChatFunction and ChatFunctionDynamic to implement a shared interface so that this can be typed as something other than <?>?